requirements/conda.txt

```diff
@@ -11,8 +11,8 @@ pickleshare
 pyzmq
 chardet>=2.0.0
 numpydoc
-pyqt5
+pyqt>=5.6
```

Comment:
Do we also want to specify <5.10 here like we have in setup.py (IIRC) since a number of bugs were reported with that?

Reply:
Right, I forgot about it.